### DISCUSSION TITLE
Remove padding from NavBar

### DISCUSSION THREAD

**Discussion by HastaCs** [heroui-inc/heroui#1683] [Q&A] [answered]
Hi, there is a way to remove the default px-6 from the Navbar component?
[Navbar](https://nextui.org/docs/components/navbar)
Thanks.

![imagen](https://github.com/nextui-org/nextui/assets/53431308/79c24700-da18-4523-a977-9c4f7d6f915d)
![imagen](https://github.com/nextui-org/nextui/assets/53431308/337cc0a9-6922-48bc-94d8-af11811a08cb)

**Comment by CaseyBlackburn** [comment, marked as answer]
`<Navbar classNames={{ wrapper: "px-0" }}>`

**Reply by HastaCs** [reply]
nice!, thank you
